### PR TITLE
new /congrats dashboard page

### PR DESCRIPTION
**Pull request by Erin007**
This PR sets up the new route and JavaScript entry point for studio.code.org/congrats which is where people who finish a Code.org Hour of Code tutorial will land. Currently, there's a placeholder component just to show that React can be rendered on the page. The url isn't linked to from anywhere and the url isn't discoverable, so I'm building piecemeal without a flag. 

screen shot 2017-10-26 at 5 17 19 pm

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -459,6 +459,7 @@ describe('entry tests', () => {
     'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
     'scriptOverview':               './src/sites/studio/pages/scriptOverview.js',
     'home/_homepage':               './src/sites/studio/pages/home/_homepage.js',
+    'congrats/_congrats': './src/sites/studio/pages/congrats/_congrats.js',
```
Comment (joshlory) [inline apps/Gruntfile.js]: Tiny nit: match indentation of other entry points?

File: dashboard/app/views/congrats/_congrats.html.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- content_for(:head) do
+  %script{src: minifiable_asset_path('js/congrats/_congrats.js')}
```
Comment (joshlory) [inline dashboard/app/views/congrats/_congrats.html.haml]: If we move this script tag to the bottom of the page we don't have to wait for `$(document).ready(showContent);` — we can call ReactDOM.render right away.

File: dashboard/app/views/congrats/index.html.haml [hunk 3]
```diff
@@ -0,0 +1 @@
+= render partial: 'congrats/congrats'
```
Comment (joshlory) [inline dashboard/app/views/congrats/index.html.haml]: Can we remove this and rename the partial to `index.html.haml`?

File: apps/src/sites/studio/pages/congrats/_congrats.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Notification from '@cdo/apps/templates/Notification';
+
+$(document).ready(showContent);
+
+function showContent() {
+
+  ReactDOM.render (
```
Comment (joshlory) [inline apps/src/sites/studio/pages/congrats/_congrats.js]: Tiny nit: extra space.

File: apps/src/sites/studio/pages/congrats/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Notification from '@cdo/apps/templates/Notification';
+
+$(document).ready(function () {
```
Comment (joshlory) [inline apps/src/sites/studio/pages/congrats/index.js:6]: Not needed if you take out `content_for(:head) do ...` below.

File: dashboard/app/views/congrats/index.html.haml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+-# Components generated by HAML which might be pulled in and displayed by the React model.
+.hiddenComponents{style: "display: none"}
+
+.clear{style: "clear: both"}
```
Comment (joshlory) [inline dashboard/app/views/congrats/index.html.haml:4]: There shouldn't be any floated elements above this view right?

File: apps/src/sites/studio/pages/congrats/index.js (hunk 5 above)
Comment (breville) [reply apps/src/sites/studio/pages/congrats/index.js:6]: How so?

**Pull request merged**